wordpress-mobile/WordPress-iOS
# Adds MediaRequestAuthenticator.

This PR implements `MediaRequestAuthenticator`.  This class takes care of authenticating media requests to the backend.

This is the first part of a larger change (#13770).

## To test:

This PR introduces the new service alongside unit tests.  It remains disconnected from the Application for the scope of this specific PR, so there's no way to test the changes within the App.

What you can test / check:

- Make sure the logic is right.  Including the logic of the unit tests.
- Let me know if you want PAW service calls to test the endpoints directly for comparison purposes with the JSONs used in the unit test.

## PR submission checklist:
